scripts/package/package.ps1

```diff
@@ -14,3 +14,9 @@ _ "$RepoRoot\packaging\windows\generatemsi.ps1" @("$Stage2Dir")
 
 header "Generating NuGet packages"
 _ "$RepoRoot\packaging\nuget\package.ps1" @("$Stage2Dir\bin", "$env:VersionSuffix")
+
+header "Generating version badge"
+$VersionBadge = "$RepoRoot\resources\images\version_badge.svg"
+(get-content $VersionBadge).replace("ver_number", "$env:DOTNET_CLI_VERSION") | set-content $VersionBadge
```

Comment:
let's write the version badge to the artifacts directory. This approach will make commiting a pain since we'll be updating a source file at build time.